common.mk

```diff
@@ -24,7 +24,7 @@ export JEST_ENV := $(call env_or_else_default,JEST_ENV)
 
 .PHONY: start-server
 start-server:
-	cellxgene launch -p $(CXG_SERVER_PORT) $(CXG_OPTIONS) $(DATASET)
+	python -m scripts.start_test_server $(PROJECT_ROOT)
```

Comment:
Don't we still want to be able to pass in an arbitrary dataset or dataset root? For local testing

Reply:
similar to @MDunitz 's suggestion, have fixed